src/classes/AFFL_AccRecordType_TEST.cls

```diff
@@ -331,4 +331,42 @@ public with sharing class AFFL_AccRecordType_TEST {
         system.assertEquals(hh.Id, markUpdated.Primary_Household__c);
         system.assertEquals(null, markUpdated.Primary_Organization__c);
     }
+
+    @isTest
+    public static void duplicateAffiliationCheck() {        
+        setup();
+
+        // Insert a business account
+        Account bizAcc = UTIL_UnitTestData_API.getMultipleTestAccounts(1, orgRecTypeID)[0];
+        insert bizAcc;
+
+        // Insert a contact with business account as Primary business organization
+        Contact contact = UTIL_UnitTestData_API.getContact();
+        insert contact;
+        
+        //Verify default household account has been created for the contact
+        contact = [SELECT AccountId,  Primary_Organization__c FROM Contact WHERE Id = :contact.Id];
+        contact.Primary_Organization__c = bizAcc.id;
+        update contact;
+        
+        //Verify 1 affiliation is created
+        List<Affiliation__c> affiliationList = new List<Affiliation__c>();
```

Comment:
@skosgi1 Minor comment, but lines 353 and 354 could be combined to just `List<Affiliation__c> affiliationList = [SELECT ...];`